website/content/docs/templates/hcl_templates/blocks/build/hcp_packer_registry.mdx

```diff
@@ -46,9 +46,10 @@ Some nice description about the image which artifact is being published to HCP P
     build_labels = {
       "xcode"   = "11.3.0"
       "version" = "Big Sur"
-  }
+    }
```

Comment:
Not sure if my suggested change will be indented correctly, but the closing bracket for hcp_packer_registry should be here
```suggestion
    }
  }
```